pallets/admin-utils/src/lib.rs

```diff
@@ -66,10 +66,10 @@ pub mod pallet {
     pub enum Error<T> {
         /// The subnet does not exist, check the netuid parameter
         SubnetDoesNotExist,
-        /// The max allowed validator number to be set is larger than threshold
-        StorageValueOutOfRange,
-        /// The maximum allowed UIDs is out of boundary, it not allowed
-        MaxAllowedUIdsNotAllowed,
+        /// The max allowed validator number to be set is larger than max allowed UIDs  
+        MaxValidatorsLargerThanMaxUIds,
+        /// The maximum allowed UIDs is less than UIDs already in the subnet
+        MaxAllowedUIdsLessThanCurrentUIds,
```

Comment:
"max allowed UIDs" is not the same as "max allowed validators". basically, when set a new max allowed validators, we need consider tow boundaries. 
1. max validator must be less than max UIDs. each validator also need a UID. so it doesn't make sense to set it too big. 
2. max validator must be larger than current registered UIDs. because each node could be the validators. 
I just get the logic from the code. do you think the enum string is appropriate for both boundaries.

Reply:
would be great to include some version of this clarification in the doc comment 👍🏻

Reply:
How about this? 

For `MaxValidatorsLargerThanMaxUIds`:  The maximum number of subnet validators must be less than the maximum number of allowed UIDs in the subnet.

For `MaxAllowedUIdsLessThanCurrentUIds`:  The maximum number of subnet validators must be more than the current number of UIDs already in the subnet.